### DISCUSSION TITLE
Thermal properties of buildings in SUEWS

### DISCUSSION THREAD

**Discussion by phemiobe** [UMEP-dev/UMEP#581] [Q&A] [answered]
Hello, I have just got this importatnt question from reviewers. any clue would be appreciated. How are the distinct thermal properties of different building materials considered in SUEWS?

**Comment by sunt05** [comment]
Please provide the following:

- What is the context of the reviewer's question?
- Which thermal properties are in question?

**Comment by phemiobe** [comment]
Thank you I was trying to justify why we have statistically significant variations in the simulated energy fluxes, air temperature and LST from one local climate zone to the other within the city. I mentioned that this could be also be related to different in thermal properties of the building materials e.g corrugated iron sheet  in informal urban areas and asbestos in other areas..

Thermal properties e.g conductivity, emissivity, specific heat capacity of building materials....

**Comment by sunt05** [comment, marked as answer]
Some of the thermal properties you've mentioned – such as conductivity and heat capacity – aren't explicitly considered in SUEWS. They are, instead, implicitly incorporated within the OHM coefficients and vary among different land cover types. Only albedo and emissivity are directly accounted for.

**Comment by suegrimmond** [comment]
definitely part of the OHM coefficients consideration - 

Look publications  that show OHM coefficients can be derived

Arnfield and Grimmond 

Meyn and Oke

But in the end it depends on what parameters you used and why

**Comment by sunt05** [comment]
Since this isn't an issue but more of a query, it's better suited for a "discussion" – I'll move it to the UMEP community discussion channel later.

**Comment by phemiobe** [comment]
Thank you both for the clarification
